skimage/restoration/_denoise.py

```diff
@@ -264,41 +264,53 @@ def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
 def denoise_tv_bregman(image, weight=5.0, max_num_iter=100, eps=1e-3,
                        isotropic=True, *, channel_axis=None,
                        multichannel=False):
-    """Perform total-variation denoising using split-Bregman optimization.
+    r"""Perform total variation denoising using split-Bregman optimization.
 
-    Total-variation denoising (also know as total-variation regularization)
-    tries to find an image with less total-variation under the constraint
-    of being similar to the input image, which is controlled by the
-    regularization parameter ([1]_, [2]_, [3]_, [4]_).
+    Given :math:`f` a noisy image (input data),
+    total variation denoising (also known as total variation regularization)
+    aims to find an image :math:`u` with less total variation than :math:`f`,
+    under the constraint that :math:`u` remain similar to :math:`f`.
+    This can be expressed by the Rudin--Osher--Fatemi (ROF) minimization
+    problem:
+
+    .. math::
+
+        \min_{u} \sum_{i=0}^{N-1} \left( \left| \nabla{u_i} \right| + \frac{\lambda}{2}(f_i - u_i)^2 \right)
```

Comment:
Good question, I've definitely hesitated. So you think the higher-level stuff should be in the main description, and the technical details under Notes? I did it the other way around because we need to be specific when describing the parameters (cf. original issue).

Reply:
I'd leave this in the description; it explains what the function does.

Reply:
Thanks for your input, @stefanv. Explicit is better than implicit!